### PR TITLE
Methylcoder

### PR DESCRIPTION
**Pull request by ArneKr**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/methylcoder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: methylcoder
+  version: "1.0"
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+source:
+  fn: brentp-methylcode-59ac293.tar.gz
+  url: https://github.com/brentp/methylcode/tarball/master
```
Comment (bgruening) [inline recipes/methylcoder/meta.yaml]: Please download a specific commit as tarball - the master branch can change.

**Comment by bgruening** [comment]
I also changed the python pinning. The skip in the build section should be enough.

**Comment by ArneKr** [comment]
@bgruening I can´t believe that is missed that :(. I saw that somebody added methylcoder a while back to bio.tools and noticed that it had no corresponding package. Guess it would be better to just refer to bwa-meth on the bio.tools page instead of adding an obsolete package here.

**Comment by bgruening** [comment]
@ArneKr yeah, lets add some note to bio.tools instead :)
Can we close this PR?

**Pull request closed without merging**